src/heist_path/LobbyGotoLG1Path.py

```
@@ -0,0 +1,269 @@
+        self.send_key('w', down_time=0.32)
+        self.wait_and_interact(direction="w", is_lock=True, time_out=6.4)
+        try:
+            if self.wait_and_interact(direction=" ", is_lock=False, time_out=1.0):
+                self.send_key('s', down_time=0.12)
+                self.sleep(0.20)
+                self.send_key('e', down_time=2.42)
+                self.send_key('w', down_time=0.32)
+                self.wait_and_interact(direction="w", is_lock=True)
+                pass
+        except Exception as e:
+            pass
```

Ruff (0.15.13)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)